crates/debug/src/transform/expression.rs

```diff
@@ -604,6 +602,22 @@ where
                     // Don't re-enter the loop here (i.e. continue), because the
                     // DW_OP_deref still needs to be kept.
                 }
+                Operation::Shr { .. } | Operation::Shra { .. } => {
```

Comment:
Do we need to include `Shr` here?

Reply:
Yes, I have seen dirt bits ending up in the semantic operand with `lldb`.

Reply:
Well, that may be in fact an indication that reading of WASM arguments/locals is not hygienic. :-( can you check that?

Reply:
it might be the case, though I expect DWARF expression not to know that, or the fact that register data is 16, 32 or 64 bit, and be proactive to mask only needed bits. We can keep Shr here for "symmetry".

Reply:
Okay, I'll keep an eye on this. If the _dirt issue_ happens to pop up elsewhere, I'll file a bug. Then we can revisit here.